lib/std/fmt.zig

```diff
@@ -456,6 +456,18 @@ pub fn formatType(
                 }
                 if (ptr_info.child == u8) {
                     return formatText(value, fmt, options, writer);
+                } else if (fmt.len > 0 and ((fmt[0] == 'v') or (fmt[0] == 'V'))) {
+                    try format(writer, "[", .{});
+                    var i: usize = 0;
+                    for (value) |one| {
```

Comment:
```suggestion
                    for (value) |elem, i| {
```

No need to evaluate the `i` outside.